autospotting.go

```diff
@@ -3,12 +3,11 @@ package main
 import (
 	"encoding/json"
 	"flag"
-	"fmt"
 	"log"
 	"os"
 
-	autospotting "github.com/cristim/autospotting/core"
 	lambda "github.com/eawsy/aws-lambda-go/service/lambda/runtime"
+	autospotting "github.com/cristim/autospotting/core"
```

Comment:
I think our editors will clash here, we have to find a way to make them agree on the import order.